Specs/Number/English/NumberRangeModel.json

```diff
@@ -805,4 +805,60 @@
     }
   ]
 }
+,
+{
+  "Input": "How about 2 in 5 or more",
+  "NotSupported": "javascript, python, java",
+  "Results": [
+    {
+      "Text": "2 in 5 or more",
+      "TypeName": "numberrange",
+      "Resolution": {
+        "value": "[0.4,)"
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "How about more than 2 in 5",
+  "NotSupported": "javascript, python, java",
+  "Results": [
```

Comment:
Add a "Comment" attribute describing this is the decision.
"As this can be ambiguous, by design the interpretation is left-to-right and the range is extracted first."